script/build.js

```diff
@@ -129,6 +129,7 @@ if (options.buildtype === 'production') {
   ignoreList.push('veteran-id-card/how-to-upload-photo.md');
   ignoreList.push('education/opt-out-information-sharing.md');
   ignoreList.push('education/complaint-tool.md');
+  ignoreList.push('disability-benefits/apply/form-526-disability-claim.md');
```

Comment:
@erikphansen metal smith is the core driver of our build process. the static content and styling is built before webpack takes over for the js apps. Currently this array is the only way to exclude static content pages from the site.